test/server.js

```diff
@@ -91,13 +119,15 @@ describe("server", function() {
                 request.put({
                     url: todoListUrl + "/0",
                     json: {
-                        title: "This is an updated TODO item"
+                        title: "This is an updated TODO item",
```

Comment:
it's tested with the additions for task_5 (7ff2029), I could put in extra cases?